### ISSUE TITLE
Inconsistencies in using HOMDEL/DeepDel

### ISSUE THREAD

**Issue by kalletlak** [cBioPortal/cbioportal#6923]
Different naming conventions for homozygous deletion is followed in different pages.
Examples
1. In OQL -> `HOMDEL`
2. Study-view CNA table -> `DEL`
3. Patient-view Copy Number Alterations table -> `DeepDel`
4. Results page Mutations tab -> `DeepDel`
4. Patient view Mutations table -> `DeepDel`
5. Comparison page copy-number tab -> `DeepDel`
6. Oncoprint -> `HOMODELETED` (in tooltip)

For hetloss,
1. In OQL -> `HETLOSS`
2. Results page Mutations tab -> `ShallowDel`
3. Patient view Mutations table -> `ShallowDel`
4. Oncoprint -> `ShallowDel` (in tooltip)

**Comment by jjgao** [comment]
@cBioPortal/product Should we change everything to `DeepDel`? (We can support both `HOMDEL` and `DeepDel` in oql for backward compatibility.)

**Comment by tmazor** [comment]
Sounds good to me. But then we should also change HETLOSS in OQL (and anywhere else it might appear).

**Comment by schultzn** [comment]
Agreed. But it would be nice if we could still support the old terms (maybe in an undocumented way).

**Comment by jjgao** [comment]
> Agreed. But it would be nice if we could still support the old terms (maybe in an undocumented way).

Supporting the old terms in an undocumented way sounds good to me.

**Comment by jjgao** [comment]
> we should also change HETLOSS in OQL (and anywhere else it might appear).

@tali Good point. @kalletlak I have updated the description to include hetloss.

@schultzn @tmazor I don't think shallowDel is a good term. This would be an opportunity to change the term to something different. But I couldn't think of better ones. Any thoughts?

**Comment by tmazor** [comment]
Maybe one of these instead of Shallow? @jjgao @schultzn 
Partial
Moderate
Intermediate

**Comment by jjgao** [comment]
It seems that "partial" is used to indicate "part of chromosome" in the copy number field (https://en.wikipedia.org/wiki/Aneuploidy#Partial_aneuploidy). In our case, "partial" may also be interpreted as partial deletion of a gene (e.g. c-terminal deletion).

And it looks like "Shallow deletion" is being used in the field now (maybe mainly b/c of us): https://www.google.com/search?q=shallow+deletion, including this Coursera course: https://www.coursera.org/lecture/data-genes-medicine/copy-number-alterations-1Ybdq

Maybe we can keep it? @tmazor @schultzn

**Comment by tmazor** [comment]
Wow, we've made an impact! I'm ok with keeping shallow - I don't feel super strongly one way or the other. If shallow has spread beyond the site, it probably does make sense to keep it.

And if we do decide to change it, good point about "partial" - definitely not a good choice for a replacement.

**Comment by schultzn** [comment]
Matthew Meyerson and Andy Cherniak originally proposed shallow, I believe, and they felt very strongly about this. Most of our copy-number calls at the time were based on GISTIC, and they knew that they could not be certain about the difference between heterozygous and homozygous. 

I also propose that we keep the term.

**Comment by jjgao** [comment]
Sounds good. @kalletlak we are ready to move this forward. It's not urgent, but could you help implement this when you have the bandwidth?

**Comment by jjgao** [comment]
Maybe @jtquach1 can help with this one? cc @Luke-Sikina

**Comment by jtquach1** [comment]
> Maybe @jtquach1 can help with this one? cc @Luke-Sikina

yes, I can help

**Comment by jtquach1** [comment]
> Different naming conventions for homozygous deletion is followed in different pages.
> Examples
> 
> 1. In OQL -> `HOMDEL`
> 2. Study-view CNA table -> `DEL`
> 3. Patient-view Copy Number Alterations table -> `DeepDel`
> 4. Results page Mutations tab -> `DeepDel`
> 5. Patient view Mutations table -> `DeepDel`
> 6. Comparison page copy-number tab -> `DeepDel`
> 7. Oncoprint -> `HOMODELETED` (in tooltip)
> 
> For hetloss,
> 
> 1. In OQL -> `HETLOSS`
> 2. Results page Mutations tab -> `ShallowDel`
> 3. Patient view Mutations table -> `ShallowDel`
> 4. Oncoprint -> `ShallowDel` (in tooltip)

New areas changed:

- Oncoprint > Download > Tabular (homdel and homdel_rec to deepdel and deepdel_rec)
![image](https://user-images.githubusercontent.com/33106214/105399652-ed374380-5bf1-11eb-997d-579b3a32e730.png)

- Download > Type of Genetic Alterations Across All Samples (table row value under `gene name : alteration type`)
![image](https://user-images.githubusercontent.com/33106214/105399814-1f48a580-5bf2-11eb-8468-cb1807e9f70e.png)
